lib/popup.html

```diff
@@ -5,22 +5,39 @@
      <link rel="stylesheet" href="../assets/popup.css">
   </head>
   <body>
-    <header>
-      <img src="../assets/white_logo.png" alt="Qowala logo"/>
-      <span>Hey, it's your friends at Qowala!</span>
-      <div></div>
-    </header>
-    <div class="non-measure-display">
-      <h3>This website is not tracked!</h3>
-    </div>
-    <div class="measure-display">
-      <h3>You've spent...</h3>
-      <div class="time-spent"></div>
-      <div class="parameters">
-        On <select class="parameters__dropdown network"></select>
-        <select class="parameters__dropdown date"></select>
+    <main id="app">
+      <header>
+        <img src="../assets/white_logo.png" alt="Qowala">
+        <hr>
+      </header>
+      <div v-if="tracked">
+        <p class="text secondary">
+          You've spent…
+        </p>
+        <div class="time-spent">
+          {{ timeSpent }}
+        </div>
+        <div class="parameters">
+          On
+          <select v-model="hostname" class="parameters__dropdown network">
+            <option v-if="noData" disabled :value="hostname">Please select another site</option>
```

Comment:
Well, it is quite disturbing to see a message like "No data on x" and nothing in the website dropdown. I think it clearly shows the user what to do to get rid of this error.